glyphs2fontir/src/source.rs

```diff
@@ -810,7 +810,7 @@ mod tests {
             let mut loc = UserLocation::new();
             loc.set_pos("Optical Size", UserCoord::new(*opsz));
             loc.set_pos("Weight", UserCoord::new(*wght));
-            let loc: _ = loc;
+            let loc = loc;
```

Comment:
Is this actually necessary?

Reply:
Not really; I just like that it clearly ends mutable time.